StripeApplePay/StripeApplePay/Source/ApplePayContext/STPApplePayContext.swift

```diff
@@ -359,16 +381,15 @@ public class STPApplePayContext: NSObject, PKPaymentAuthorizationControllerDeleg
     }
 
     func _end() {
-        if let authorizationController = authorizationController {
-            objc_setAssociatedObject(
-                authorizationController,
-                UnsafeRawPointer(&kApplePayContextAssociatedObjectKey),
-                nil,
-                .OBJC_ASSOCIATION_RETAIN_NONATOMIC
-            )
-        }
-        authorizationController = nil
+        objc_setAssociatedObject(
```

Comment:
Before, if `_end()` was (erroneously) called multiple times, we wouldn't call `objc_setAssociatedObject` twice, but now that `authorizationController` no longer nullable, it's possible to call `objc_setAssociatedObject`. Is there any protection we can add to avoid this?

Reply:
It's safe to call `objc_setAssociatedObject` multiple times, tried it out in a test without issue.